tests/vectordb/test_chroma_db.py

```diff
@@ -1,9 +1,10 @@
 import os
 import shutil
-from unittest.mock import patch
+from unittest.mock import MagicMock, patch
```

Comment:
Can you please use pytest-mock instead here? We want to move away from unittest.

Reply:
Done